### ISSUE TITLE
A simple extension to display a popup when you click on the Add To Cart button of your products.

### ISSUE THREAD

**Issue by poehaldur** [Crocoblock/suggestions#548]
For woobuilder, can you make a functionality, that notifies via popup or opens Cart dropdown in menu automatically when product is placed in cart?

For example OceanWP has this: https://oceanwp.org/extension/ocean-woo-popup/

I would love to have a way to give options to users after they have added product in cart. Currently when you add product to cart, you get traditional woocommerce notification bar on top of product page.

So basically. The idea is: When user adds product to cart, he sees either a popup or automatically opened cart dropdown:
![woo-popup-screen](https://user-images.githubusercontent.com/32192439/65447950-61b47380-de40-11e9-8645-f5a1a7bfe6a6.png)
![download (5)](https://user-images.githubusercontent.com/32192439/65447951-624d0a00-de40-11e9-99fa-49b87c3f17a7.png)

**Comment by yoske25** [comment]
Would Love that also :) thanks!

**Comment by UraraReika** [comment]
Done. Next Release.

**Comment by iamdanbarrett** [comment]
So where is the option for this? I have just updated the plugin

**Comment by Floydtm** [comment]
@poehaldur, Hi! You can check out this [tutorial ](https://crocoblock.com/knowledge-base/articles/jetwoobuilder-how-to-display-a-products-cart-in-a-popup-based-on-jetpopup/)

**Comment by ninjamasterchris** [comment]
Can you please also make this available at the Add to Cart widget for the Single Product Page.

**Comment by redifmail** [comment]
Kindly make it open via single product page add to cart. Just like Elementor.
